cli/index.js

```diff
@@ -18,6 +18,7 @@ import chalk from "chalk";
 import { App } from "../client/src/app";
 import render from "./render";
 import { fixSyntaxHighlighting } from "./syntax-highlighter";
+import { normalizeURLs } from "./browser-compatibility-table";
```

Comment:
What do you think about using an alias import here? *Within* `browser-compatibility-table.js` the name makes perfect sense, but out here in this file the name is wildly generic.

Reply:
Yeah i agree, I'll come up with something. 👍